### PR TITLE
add: docmost

### PR DESCRIPTION
**Pull request by TornaxO7**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions. (didn't try, but there are good comments about it in the selfhost subreddit)
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/docmost.yml
Comment (Rabenherz112) [inline software/docmost.yml:4]: > https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

```suggestion
description: "Collaborative wiki and documentation software (alternative to Confluence, Notion)."
```

File: software/docmost.yml
Comment (Rabenherz112) [inline software/docmost.yml:8]: [Install guide](https://docmost.com/docs/installation) shows main installation method to be docker

```suggestion
platforms:
  - Docker
  - Nodejs
```

File: software/docmost.yml
Comment (Rabenherz112) [inline software/docmost.yml:10]: I am not sure, if this tag is needed as it seems primarily to be a wiki (and therefore the wiki tag should be enough already) but let's wait for some other reviewers on that

Comment (kokomo123) [reply software/docmost.yml:10]: Seeing as knowledge management tools tag is related to wikis, yeah this is redundant.

Comment (TornaxO7) [reply software/docmost.yml]: done

**Review by kokomo123 — APPROVED** [review]
Seeing as the appropriate changes have been made, LGTM pending further thoughts from collaborators. Thanks for the fixes, and thanks for your contribution 👍🏻

**Comment by TornaxO7** [comment]
You are welcome

peepoAwesome

**Review by TheGiraffe3 — APPROVED** [review]

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**
